source/includes/fundamentals/code-examples/builders.cs

```diff
@@ -12,25 +12,41 @@ public static void Main(string[] args)
     {
         Setup();
 
-        // Sample builders definitions
+        // Creates a filter for all documents with a "season" value of "Spring"
         var filter = Builders<Flower>.Filter.AnyEq(flower => flower.Season, "spring");
+
+        // Creates a projection to include the "name" and "price" fields, while
+        // excluding the "id" field
         var projection = Builders<Flower>.Projection.Include("Name").Include("Price").Exclude("Id");
+
+        // Creates a sort object that sorts results by ascending price, and, in
+        // the case the multiple documents have the same price, sorts in reverse
+        // alphabetical order by category
         var sort = Builders<Flower>.Sort.Ascending("Price").Descending("Category");
 
         Console.WriteLine("Finding documents...");
+
+        // Finds all documents that match the filter sorted according to the
+        // sort object, including on the fields defined by the projection
         var result = _flowerCollection.Find(filter).Sort(sort).Project(projection).toList();
 
         Console.WriteLine(result.ToJson());
 
+        // Creates an update object for that will multiply the price value by 0.9
         var update = Builders<Flower>.Update.Mul("Price", 0.9);
+
+        // Applies the update object instructions to the first document that
+        // matches the filter
```

Comment:
S: just a little more concise

```suggestion
        // Updates the first document that matches the filter
```